workorders.lic

```diff
@@ -294,6 +299,31 @@ class WorkOrders
     end
   end
 
+  def order_cloth(stock_room, stock_needed, stock_number)
```

Comment:
This looks like it's the same as `order_yarn` except for the noun. Could that be a parameter passed into a common function?